signin.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>회원가입</title>
+    <link rel="stylesheet" href="./src/sign.css" />
+</head>
+<body>
+    <header>
+        <a class="header-logo" href="/index.html">
+            <img src="./images/logo.png" alt="홈으로 연결된 Linkbrary 로고" />
+        </a>
+        <p class="header-message">
+            회원이 아니신가요?<a class="header-link" href="signup.html">회원 가입하기</a>
+        </p>
+    </header>
+    <div class="sign-box">
+        <form class="sign-form">
+          <div class="sign-inputs">
+            <div class="sign-input-box">
+              <label class="sign-input-label">이메일</label>
+              <input class="sign-input" />
+            </div>
+            <div class="sign-input-box sign-password">
+              <label class="sign-input-label">비밀번호</label>
+              <input class="sign-input" type="password" />
```

Comment:
label 요소로 input 요소를 감싸거나 [for 속성](https://developer.mozilla.org/ko/docs/Web/HTML/Element/label#for)을 지정하여 연관된 폼 제어 요소과의 관계를 나타낼 수 있어요.

label과 input을 연결시키면 label 클릭 시 폼 제어 요소가 자동 포커스되고 스크린 리더가 input에 해당하는 label을 읽어 접근성이 더욱 좋아집니다.

```suggestion
              <label for="password" class="sign-input-label">비밀번호</label>
              <input id="password" class="sign-input" type="password" />
```